modules/apps/object/object-test/src/testFunctional/tests/Object.testcase

```diff
@@ -21,6 +21,143 @@ definition {
 		}
 	}
 
+	@description = "LPS-TEST - Verify if it is possible to create a Custom Object"
+	@priority = 5
+	test CanCreateCustomObject {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 01",
+			objectName = "CustomObject01",
+			pluralLabelName = "Custom Objects 01");
+
+		ObjectAdmin.openObjectAdmin();
+
+		Refresh();
+
+		Pause(locator1 = 3000);
+
+		AssertElementPresent(
+			key_label = "Custom Object 01",
+			locator1 = "ObjectAdmin#SELECT_OBJECT_FIELD");
+	}
+
+	@description = "LPS-TEST - Verify it is possible to create a String field"
+	@priority = 5
+	test CanCreateStringFieldInCustomObject {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 01",
+			objectName = "CustomObject01",
+			pluralLabelName = "Custom Objects 01");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object 01");
+
+		ObjectAdmin.goToFieldsTab();
+
+		LexiconEntry.gotoAdd();
+
+		Pause(locator1 = 2000);
+
+		Type(
+			locator1 = "ObjectField#LABEL_INPUT",
+			value1 = "Custom String Field 01");
+
+		ObjectField.selectType(fieldType = "Text");
+
+		Pause(locator1 = 2000);
+
+		PortletEntry.save();
+
+		Refresh();
+
+		Pause(locator1 = 2000);
+
+		AssertElementPresent(
+			key_label = "Custom String Field 01",
+			locator1 = "ObjectAdmin#SELECT_OBJECT_FIELD");
+	}
+
+	@description = "LPS-TEST - Verify it is possible to add a field after the Object is published and submit entries to it"
+	@priority = 5
+	test CanAddFieldAfterObjectPublishedAndSubmitEntries {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 777",
+			objectName = "CustomObject777",
+			pluralLabelName = "Custom Objects 777");
+
+		Navigator.openURL();
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object 777");
+
+		ObjectAdmin.goToFieldsTab();
+
+		ObjectAdmin.addObjectFieldViaUI(
+			fieldLabel = "Full Name",
+			fieldType = "Text");
+
+		Refresh();
+
+		ObjectAdmin.assertObjectField(
+			fieldLabel = "Full Name",
+			fieldType = "Text");
+
+
+		ObjectAdmin.goToDetailsTab();
+
+		CreateObject.selectPanelCategoryKey(panelCategoryKey = "Control Panel > Object");
+
+		ObjectAdmin.publishObjectViaUI();
+
+		Navigator.openURL();
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object 777");
+
+		ObjectAdmin.goToFieldsTab();
+
+		ObjectAdmin.addObjectFieldViaUI(
+			fieldLabel = "Phone Number",
+			fieldType = "Long Integer");
+
+		Refresh();
+
+		ObjectAdmin.assertObjectField(
+			fieldLabel = "Phone Number",
+			fieldType = "LongInteger");
+
+		
+
```

Comment:
Why are there empty spaces here? For these and other fixes, you must do the following:

1. After making the commits, go to the "dev/projects/liferay-portal/portal-impl" folder and execute the following command: ant format-source-current-branch. You can go to the ".bashrc" folder and check if there is an alias for it.
2. Wait for the command to run. It may last for a few minutes.
3. Check the corrections that were made and commit them. For example: "LPS 0000 SF".

Reply:
ok, I just didn't understand the third step very well.

Reply:
The source format will make corrections to your code if there are syntax errors. Like for example these vague lines in your code.

Reply:
After running the command, wait for the execution and "git status" to check if corrections were made.